components/automate-ui/src/app/pages/policy/add-members/policy-add-members.component.html

```diff
@@ -14,27 +14,84 @@
     <div *ngIf="!loading">
       <chef-modal label="member-expression-label" class="member-expression-modal" [visible]="modalVisible" (close)="closeModal()">
         <h2>Add Member Expression</h2>
-        <div id="left-align">
+        <div id="modal-content-container">
           <form [formGroup]="expressionForm">
+
+              <chef-form-field id="create-type">
+                <label>
+                  <span class="label">Type<span aria-hidden="true">*</span></span>
+                  <select id="expression-type-dropdown"
+                          ngDefaultControl 
+                          formControlName="type">
+                    <option value='USER'>User</option>
+                    <option value='TEAM'>Team</option>
+                    <option value='TOKEN'>Token</option>
+                    <option value='*'>All members (*)</option>
+                  </select>
+                </label>
+                <chef-error *ngIf="expressionForm.get('type').hasError('required') && expressionForm.get('type').touched">
+                  Type is a required field.
+                </chef-error>
+              </chef-form-field>
+
+              <chef-form-field 
+                id="create-identity" 
+                *ngIf="showInputs('identity')"
+                [@dropInAnimation]="{ value: triggerValue, params: { height: 74.6875 }}" >
```

Comment:
params let us conditionally pass in a variable for the animation.  Since we do not have to worry too much about screen size changes, I've chose to pass in the exact height the element sits at naturally, this helps to reduce any animation jitters.